src/ray/raylet/main.cc

```diff
@@ -236,6 +224,13 @@ int main(int argc, char *argv[]) {
         server->Start();
       }));
 
+  // Initialize stats.
```

Comment:
Currently, we share the io_service of each component to enable asynchronous metrics report. This will be same for all other cpp components. Please lmk if you are concerned about that it affects performance.

Reply:
Should be fine for now